deploy/scripts/src/mobile/utils/url.ts

```diff
@@ -0,0 +1,28 @@
+import { URL } from './k6/url'
+import { config } from './config'
+
+const OAUTH_ROUTE = '/dca/oauth2'
+
+export function getUrl (
+  path: string,
+  base: string,
+  query?: Record<string, string>
+): string {
+  const url = new URL(path, base)
+
+  if (query != null) {
+    Object.entries(query).forEach(([key, value]) => {
+      url.searchParams.set(key, value)
+    })
+  }
+
+  return url.toString()
+}
+
+export function getFrontendUrl (path: string, query?: Record<string, string>): string {
```

Comment:
could we name this to something more specific? It sounds to me like it's fetching the config.frontendUrl, but it's actually building a request

Reply:
Updated these function names to `buildUrl`, `buildFrontendUrl` and `buildBackendUrl`

Reply:
nice - looks good